docs/metrics.rst

```diff
@@ -41,6 +41,13 @@ Add the following lines to your configuration file e.g. ``airflow.cfg``
     statsd_port = 8125
     statsd_prefix = airflow
 
+If you want to avoid send all the available metrics to StatsD, you can configure a whitelist of prefixes to send only
+the metrics that start with the elements of the list:
+
+.. code-block:: bash
```

Comment:
```suggestion
.. code-block:: none
```

Reply:
Hi @mik-laj, I just checked and currently, there is no `.. code-block:: init` in the codebase. Do you think it's a good idea to start that?

Reply:
`.. code-block:: ini` was probably what he meant.

Reply:
Oh now makes sense! I'll do it.